crates/cli/src/lib.rs

```diff
@@ -276,6 +277,23 @@ pub fn build_app<'a>() -> Command<'a> {
                     .required(true)
             )
         )
+        .subcommand(Command::new(CMD_GEN_DUMMY_LIB)
+            .about("Generate a dummy shared library that can be used for linking a platform binary")
```

Comment:
Can we add a note about how this is only for `--linker=legacy`, and will go away when `--linker=legacy` goes away? 😄

Reply:
How does this sound:

```
Generate a stubbed shared library that can be used for linking a platform binary.
The stubbed library has prototypes, but no function bodies.

Note: This command will be removed in favor of just using `roc build` once all platforms support the
surgical linker
```